test/Concurrency/transfernonsendable_ownership.swift

```diff
@@ -105,3 +105,19 @@ func testInOutError(_ x: inout Klass) async {
   // expected-note @-1 {{sending global actor 'CustomActor'-isolated 'x' to main actor-isolated global function 'transferToMain'}}
   _ = consume x // expected-note {{consumed here}}
 } // expected-note {{used here}}
+
+actor ActorTestCase {
+  let k = Klass()
+
+  // TODO: This crashes the compiler since we attempt to hop_to_executor to the
+  // value that is materialized onto disk.
+  //
```

Comment:
While doing this list of look through, I wanted to see how having consuming/borrowing as actor self worked... and I found in the consuming case we crashed. I am going to fix that in a different PR. But I thought it was worth putting in tree.

Reply:
Nice catch, thanks for followup then as well :)